app/views/feedbacks/new.html.erb

```diff
@@ -74,6 +74,7 @@
     <legend><%= ts("Send Your Feedback") %></legend>
     <%= f.hidden_field :user_agent, value: request.env["HTTP_USER_AGENT"] %>
     <%= f.hidden_field :ip_address, value: request.remote_ip %>
+    <%= f.hidden_field :rollout, value: @feedback.rollout_string %>
```

Comment:
Yeah, I figure we don't care so much because like you said, it's already possible to do that with the IP and user agent. If someone changes one of those, all they accomplish is possibly getting themselves an unhelpful answer that doesn't pertain to them.